db/migrate/20180626160422_add_upstream_pool_id_to_katello_pool.rb

```diff
@@ -0,0 +1,9 @@
+class AddUpstreamPoolIdToKatelloPool < ActiveRecord::Migration[5.1]
```

Comment:
I think this requires an installer update to make sure that the `katello:reimport` task is run otherwise all the upstream_pool_ids will be nil

Reply:
I believe a reimport is run (at least for subs/pools) on upgrade - @jlsherrill or @ehelms can you confirm?

Reply:
we don't currently do this on every upgrade, we only did it once at some point.  we'd want to add an upgrade task for 3.7 to do that.